std_msgs/Bool refine_prediction
std_msgs/Bool refine_refinement
std_msgs/Byte number
std_msgs/Bool gender
std_msgs/Byte age